alpha/external-web/app/eligibility-client.js

```diff
@@ -32,5 +32,14 @@ exports.update = function (id, claimant, callback) {
 
 // Takes a string and wraps it as a Mongo ObjectID.
 exports.mongoId = function (id) {
-  return new mongo.client.ObjectID(id)
+  if (exports.isValidMongoId(id)) {
+    return new mongo.client.ObjectID(id)
+  }
+  throw new Error({
+    error: 'Invalid Mongo ID'
+  })
+}
+
+exports.isValidMongoId = function (id) {
```

Comment:
I don't think we should be explicitly stating our use of mongo in the eligibility client, use generic names like isValidReference